lighthouse-core/audits/uses-rel-preconnect.js

```diff
@@ -35,6 +35,11 @@ const UIStrings = {
    * */
   crossoriginWarning: 'A preconnect <link> was found for "{securityOrigin}" but was not used ' +
     'by the browser. Check that you are using the `crossorigin` attribute properly.',
+  /**
```

Comment:
Don't need the `@description` tag if it is the only tag.

```suggestion
  /** A warning message that is shown when found more than 3 preconnected links */
```

plus delete l#39 and l#40